src/app/content/search/reducer.ts

```diff
@@ -13,6 +13,7 @@ export const initialState: State = {
   results: null,
   selectedResult: null,
   sidebarOpen: false,
+  userSelectedResult: false,
```

Comment:
@TomWoodward I added this to determine when the sidebar should scroll or not

Reply:
this would be removed when the autoselect on initial search is removed?